src/main/java/onseo/jerseytest/services/SummaryService.java

```diff
@@ -27,8 +33,48 @@ public UserEntity getUser(int userId) {
         return userEntityPlaceholderClient.getEntity(userId, "/users/", UserEntity.class);
     }
 
-    public Object getSummaryObject(int id) {
-        //TO BE IMPLEMENTED
-        return null;
+    public <T extends Entity> T getEntityAsync(int id, String endpointSuffix, Class<T> cls) {
```

Comment:
all of these methods can be replaced with 4 code lines with common try ... catch

Reply:
You use a lot of suffixes "/todo/" "/user/" in different places
remove "String endpointSuffix" from arguments of PlaceholderClient.getEntity(...) and create private String getSuffix(Class<T> cls) in PlaceholderClient to calculate suffix. Suffixes can be stored in application.yml and provided by PlaceholderClientProperties